cypress/e2e/signUp.cy.js

```diff
@@ -1,5 +1,16 @@
+/// <reference types="cypress" />
+const { generateUser } = require('../support/generate');
 describe('Sign Up page', () => {
+  before(() => {
+    const URL_SIGN_UP = 'https://react-redux.realworld.io/#/register?_k=usft36';
```

Comment:
its better to add baseUrl to cypressConfig or just write URL in cy.visit without variable